examples/GE-Signa-PETMR/howto.md

```diff
@@ -0,0 +1,44 @@
+# Reconstructing data from GE SIGNA PET/MR Scanner
+
+Author: Palak Wadwha
+Edits: Kris Thielemans
+
+These instructions use a  template that was created using the `create_projdata_template` utility using the following inputs:
+```
+Enter the name of the scanner : GE PET/MR Signa
+Mashing factor for views : 1
+Is the data arc-corrected?: N
+Number of the tangential positions: 357
```

Comment:
The tool only accepts numbers in the range [0 128]. 
```

Number of tangential positions[1,128 D:128]: 357

Out of bounds. Try again.
```
Missing instructions?

Reply:
turned out the name was wrong. It has to be `GE PET/MR Signa`. now corrected.